.travis.yml

```diff
@@ -1,4 +1,7 @@
-language: python
+sudo: required
```

Comment:
Are you sure that's needed? Is that because of switching to `language: java`? I thought they had rid themselves of having to specify that. It used to be necessary for Docker-utilizing builds, which this one isn't.